plugins/org.eclipse.xtext.xbase.ide/META-INF/MANIFEST.MF

```diff
@@ -8,7 +8,7 @@ Bundle-RequiredExecutionEnvironment: JavaSE-1.6
 Bundle-ActivationPolicy: lazy
 Bundle-Localization: plugin
 Require-Bundle: org.eclipse.xtext.ide,
- org.eclipse.xtext.xbase,
+ org.eclipse.xtext.xbase;visibility:=reexport,
```

Comment:
I understand that concern with bundles outside our control, but why would anybody use xbase.ide without using xbase? I thought that was the same rationale why xbase.ui exports xbase too.

If we don't want this, then we'll have to add more require bundles to the XbaseGeneratorFragment2